.github/workflows/tests-smoke.yaml

```diff
@@ -195,7 +195,9 @@ jobs:
           tar xzvf prometheus-${PROM_VERSION}.linux-amd64.tar.gz prometheus-${PROM_VERSION}.linux-amd64/promtool
           rm -f prometheus-${PROM_VERSION}.linux-amd64.tar.gz
           sudo mv prometheus-${PROM_VERSION}.linux-amd64/promtool /usr/bin
-          cat metrics.prom | promtool check metrics
+          # filter go_ metrics which do not conform to the _total convention with gauges
+          # See https://github.com/prometheus/prometheus/issues/10574
+          cat metrics.prom | grep -v 'go_' | promtool check metrics
```

Comment:
grep -v 'go_' metrics.prom can be used directly, I am guessing you want to keep it this way for easy revert once the perm fix is there.

Reply:
Ah, I just wasn't thinking about that, but sure let's go with that reasoning 👀 .